src/main/scala/org/broadinstitute/hail/driver/LogisticRegressionCommand.scala

```diff
@@ -0,0 +1,134 @@
+package org.broadinstitute.hail.driver
+
+import breeze.linalg.{DenseMatrix, DenseVector, sum}
+import org.broadinstitute.hail.Utils._
+import org.broadinstitute.hail.annotations.Annotation
+import org.broadinstitute.hail.expr._
+import org.broadinstitute.hail.methods.LogisticRegression
+import org.kohsuke.args4j.{Option => Args4jOption}
+
+object LogisticRegressionCommand extends Command {
+
+  def name = "logreg"
+
+  def description = "Test each variant for association using the logistic regression model"
+
+  class Options extends BaseOptions {
+    // FIXME: specify that must be Boolean or 0 / 1
+    @Args4jOption(required = true, name = "-y", aliases = Array("--y"), usage = "Response sample annotation")
+    var ySA: String = _
+
+    @Args4jOption(required = false, name = "-c", aliases = Array("--covariates"), usage = "Covariate sample annotations, comma-separated")
+    var covSA: String = ""
+
+    @Args4jOption(required = false, name = "-t", aliases = Array("--test"), usage = "Statistical test: `wald', `lrt', or `score'")
+    var stat: String = "wald"
+
+    @Args4jOption(required = false, name = "-r", aliases = Array("--root"), usage = "Variant annotation root, a period-delimited path starting with `va'")
+    var root: String = "va.logreg"
```

Comment:
Root is going away.  In the new model, you should have an expression in which the result of the test is available (say, through a variable called linreg) and the user can assign new annotations or compute whatever expressions they want.  See AnnotateVariantsTable `-c` option.